Content.Server/Atmos/Components/PressureProtectionComponent.cs

```diff
@@ -14,5 +14,12 @@ public sealed class PressureProtectionComponent : Component
 
         [DataField("lowPressureModifier")]
         public float LowPressureModifier { get; } = 0f;
+
+        /// <summary>
+        ///     The examine group used for grouping together examine details.
+        /// </summary>
+        [DataField("examineGroup")] public string ExamineGroup = "atmos";
+
+        [DataField("examinePriority")] public int ExaminePriority = 2;
```

Comment:
See above on the fact that a bunch of different components are going to have arbitrary integers which will cause confusion down the line (see: drawdepths, collision groups, etc.)